services/app/src/constants/playerStatuses.js

```diff
@@ -0,0 +1,5 @@
+export default {
```

Comment:
event types should be in shared library. We need to merge this PR https://github.com/ZenitechSoftware/planning-chess/pull/42 
and put common data model there. 
Might be out of scope for this PR though...